### PR TITLE
handle the release schedule only in the Github wiki

### PR DESCRIPTION
**Pull request by szaimen**
This idea came up in https://help.nextcloud.com/t/release-of-nc-24-delayed/137919/13 and below

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: admin_manual/release_schedule.rst [hunk 1]
```diff
@@ -2,17 +2,7 @@
...
-+--------+-------------------+------------------+
-|        | release date      | end of life      |
-+========+===================+==================+
-| **23** | 2021-11-30        | 2022-11          |
-+--------+-------------------+------------------+
-| **22** | 2021-07-06        | 2022-07          |
-+--------+-------------------+------------------+
-
-Find the detailed schedule for major and maintenance releases at: `GitHub <https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule>`_.
+You can find the detailed schedule for major and maintenance releases at: `GitHub <https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule>`_.
```
Comment (dartcafe) [inline admin_manual/release_schedule.rst:5]: Suggestion: Set the link to the term 'detailed schedule' instead of 'Github'

**Review by dartcafe — CHANGES_REQUESTED** [review]

File: admin_manual/release_schedule.rst (hunk 1 above)
Comment (szaimen) [reply admin_manual/release_schedule.rst]: done

**Review by dartcafe — APPROVED** [review]

**Pull request merged**